include/bluetooth/audio/audio.h

```diff
@@ -197,6 +197,37 @@ struct bt_codec_data {
 					  BT_CODEC_META_CONTEXT_RINGTONE | \
 					  BT_CODEC_META_CONTEXT_TV)
 
+enum bt_audio_location {
```

Comment:
I would like to have a reference to the assigned numbers here, to make it clear where these come from.